projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -101,6 +102,35 @@ public static function render_wrapped_html_block( $content, $parsed_block ) {
 		return $content;
 	}
 
+	/**
+	 * Add Jetpack Forms interactivity attributes to core/buttons blocks that live inside a contact form.
+	 *
+	 * @param string $content Rendered HTML of the core/button block.
+	 * @param array  $parsed_block Parsed block array.
+	 * @return string Possibly modified HTML.
+	 */
+	public static function render_submit_button( $content, $parsed_block ) {
+		$class = $parsed_block['attrs']['className'] ?? '';
+		if ( ! str_contains( $class, 'jetpack-form-submit-button' ) ) {
+			return $content;
+		}
+
+		if ( ! class_exists( '\WP_HTML_Tag_Processor' ) ) {
```

Comment:
It's better to use `::class` to get FQCN as strings here are not handled well by linters and our Phan.
```suggestion
		if ( ! class_exists( \WP_HTML_Tag_Processor::class ) ) {
```

For example, if there is a typo
```php
class_exists( '\WP_TML_Tag_Processor' )
//                 ^ Missing "H"
```

Phan won't catch that, but if you do this, Phan won't be happy

```php
class_exists( \WP_TML_Tag_Processor::class )
```